src/functions/Gaussian.cpp

```diff
@@ -16,7 +16,7 @@ namespace mrcpp {
 
 template<int D>
 Gaussian<D>::Gaussian(double a, double c, const double r[D], const int p[D]) {
```

Comment:
I'd use a CTOR list, rather than initializing in the body.

Reply:
I'm not able to use CTOR on `const double r[D]` and `const int p[D]` (I think). This constructor should probably be removed, but I am keeping it to avoid breaking any client code.